javascript/builtins/Atomics.json

```diff
@@ -6,27 +6,44 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Atomics",
           "support": {
             "webview_android": {
-              "version_added": "60"
+              "version_added": "60",
+              "version_removed": "64",
+              "notes": "Support was removed to mitigate <a href='https://www.chromium.org/Home/chromium-security/ssca'>speculative execution side-channel attacks (Chromium project)</a>."
             },
             "chrome": {
-              "version_added": "60"
+              "version_added": "60",
+              "version_removed": "64",
+              "notes": "Support was removed to mitigate <a href='https://www.chromium.org/Home/chromium-security/ssca'>speculative execution side-channel attacks (Chromium project)</a>."
             },
             "chrome_android": {
-              "version_added": "60"
+              "version_added": "60",
+              "version_removed": "64",
+              "notes": "Support was removed to mitigate <a href='https://www.chromium.org/Home/chromium-security/ssca'>speculative execution side-channel attacks (Chromium project)</a>."
             },
             "edge": {
-              "version_added": "16"
+              "version_added": false,
+              "notes": "Support was removed to mitigate <a href='https://blogs.windows.com/msedgedev/2018/01/03/speculative-execution-mitigations-microsoft-edge-internet-explorer'>speculative execution side-channel attacks (Windows blog)</a>."
             },
             "edge_mobile": {
               "version_added": null
             },
             "firefox": [
               {
-                "version_added": "55"
+                "version_added": "57",
+                "flag": {
+                  "type": "preference",
+                  "name": "javascript.options.shared_memory",
+                  "value_to_set": "true"
+                },
+                "notes": "Support was disabled by default to mitigate <a href='https://blog.mozilla.org/security/2018/01/03/mitigations-landing-new-class-timing-attack/'>speculative execution side-channel attacks (Mozilla Security Blog)</a>."
+              },
+              {
+                "version_added": "55",
+                "version_removed": "57"
               },
               {
                 "version_added": "46",
-                "version_removed": "54",
+                "version_removed": "55",
```

Comment:
Should this be a separate entry, or combined with the "57" entry?

Reply:
I think to properly record the history here, it should be separate.